test/test_file.cpp

```diff
@@ -408,3 +399,139 @@ TORRENT_TEST(stat_file)
 	TEST_CHECK(ec);
 	TEST_EQUAL(ec, boost::system::errc::no_such_file_or_directory);
 }
+
+// specificaly UNC tests
+#if TORRENT_USE_UNC_PATHS
+
+int maximum_component_length { 0 };
+bool support_hard_links { false };
+
+bool fill_current_directory_caps()
```

Comment:
It retrieves file system capabilities for current working directory. It was made skip certain test on host file system that can handle it (not to false ASSERT).